### PR TITLE
Product Template Version 2 (product-v2)

### PR DESCRIPTION
**Pull request by benji-glitsos-ga**
This PR has been moved to a new git branch: https://github.com/GeoscienceAustralia/dea-knowledge-hub/pull/306

### CODE REVIEW

**Comment by benji-glitsos-ga** [comment]
**Preview:**

* <https://pr-245-preview.khpreview.dea.ga.gov.au/data/product/dea-land-cover-landsat/?tab=overview>

File: docs/data/product/dea-mangrove-canopy-cover-landsat/_data.yaml [hunk 1]
```diff
@@ -15,8 +15,10 @@ spatial_data_type: Raster
 time_span:
   start: 1987
   end: 2022
-update_frequency: Yearly
-next_update: null
+data_updates:
+  frequency: DAILY
```
Comment (robbibt) [inline docs/data/product/dea-mangrove-canopy-cover-landsat/_data.yaml:19]:
```suggestion
  frequency: YEARLY
```

File: docs/data/product/dea-surface-reflectance-landsat-7-etm/_data.yaml [hunk 2]
```diff
@@ -15,8 +15,10 @@ spatial_data_type: Raster
 time_span:
   start: 28/05/1999
   end: 06/04/2022
-update_frequency: Daily
-next_update: null
+data_updates:
+  frequency: DAILY
+  activity: ONGOING
```
Comment (robbibt) [inline docs/data/product/dea-surface-reflectance-landsat-7-etm/_data.yaml:20]: Landsat 7 no longer collecting data (at least not for DEA)
```suggestion
  activity: NO_UPDATES
```

File: docs/data/product/dea-surface-reflectance-landsat-5-tm/_data.yaml [hunk 3]
```diff
@@ -15,8 +15,10 @@ spatial_data_type: Raster
 time_span:
   start: 16/08/1986
   end: 17/11/2011
-update_frequency: Daily
-next_update: null
+data_updates:
+  frequency: DAILY
+  activity: ONGOING
```
Comment (robbibt) [inline docs/data/product/dea-surface-reflectance-landsat-5-tm/_data.yaml:20]: Landsat 5 no longer collecting data
```suggestion
  activity: NO_UPDATES
```

File: docs/data/product/dea-surface-reflectance-nbar-landsat-5-tm/_data.yaml [hunk 4]
```diff
@@ -15,8 +15,10 @@ spatial_data_type: Raster
 time_span:
   start: 16/08/1986
   end: 17/11/2011
-update_frequency: Daily
-next_update: null
+data_updates:
+  frequency: DAILY
+  activity: ONGOING
```
Comment (robbibt) [inline docs/data/product/dea-surface-reflectance-nbar-landsat-5-tm/_data.yaml:20]:
```suggestion
  activity: NO_UPDATES
```

File: docs/data/product/dea-surface-reflectance-nbar-landsat-7-etm/_data.yaml [hunk 5]
```diff
@@ -15,8 +15,10 @@ spatial_data_type: Raster
 time_span:
   start: 28/05/1999
   end: 06/04/2022
-update_frequency: Daily
-next_update: null
+data_updates:
+  frequency: DAILY
+  activity: ONGOING
```
Comment (robbibt) [inline docs/data/product/dea-surface-reflectance-nbar-landsat-7-etm/_data.yaml:20]:
```suggestion
  activity: NO_UPDATES
```

File: docs/data/product/dea-surface-reflectance-nbart-landsat-5-tm/_data.yaml [hunk 6]
```diff
@@ -15,8 +15,10 @@ spatial_data_type: Raster
 time_span:
   start: 16/08/1986
   end: 17/11/2011
-update_frequency: Daily
-next_update: null
+data_updates:
+  frequency: DAILY
+  activity: ONGOING
```
Comment (robbibt) [inline docs/data/product/dea-surface-reflectance-nbart-landsat-5-tm/_data.yaml:20]:
```suggestion
  activity: NO_UPDATES
```

File: docs/data/product/dea-surface-reflectance-nbart-landsat-7-etm/_data.yaml [hunk 7]
```diff
@@ -15,8 +15,10 @@ spatial_data_type: Raster
 time_span:
   start: 28/05/1999
   end: 06/04/2022
-update_frequency: Daily
-next_update: null
+data_updates:
+  frequency: DAILY
+  activity: ONGOING
```
Comment (robbibt) [inline docs/data/product/dea-surface-reflectance-nbart-landsat-7-etm/_data.yaml:20]:
```suggestion
  activity: NO_UPDATES
```

File: docs/data/product/dea-surface-reflectance-oa-landsat-5-tm/_data.yaml [hunk 8]
```diff
@@ -15,8 +15,10 @@ spatial_data_type: Raster
 time_span:
   start: 16/08/1986
   end: 17/11/2011
-update_frequency: Daily
-next_update: null
+data_updates:
+  frequency: DAILY
+  activity: ONGOING
```
Comment (robbibt) [inline docs/data/product/dea-surface-reflectance-oa-landsat-5-tm/_data.yaml:20]:
```suggestion
  activity: NO_UPDATES
```

File: docs/data/product/dea-surface-reflectance-oa-landsat-7-etm/_data.yaml [hunk 9]
```diff
@@ -15,8 +15,10 @@ spatial_data_type: Raster
 time_span:
   start: 28/05/1999
   end: 06/04/2022
-update_frequency: Daily
-next_update: null
+data_updates:
+  frequency: DAILY
+  activity: ONGOING
```
Comment (robbibt) [inline docs/data/product/dea-surface-reflectance-oa-landsat-7-etm/_data.yaml:20]:
```suggestion
  activity: NO_UPDATES
```

File: docs/data/product/dea-surface-reflectance-oa-landsat-9-oli-tirs/index.rst~HEAD
Comment (robbibt) [inline docs/data/product/dea-surface-reflectance-oa-landsat-9-oli-tirs/index.rst~HEAD:1]: Is this meant to be here?

File: docs/data/product/dea-surface-reflectance-oa-landsat-9-oli-tirs/index.rst~benji
Comment (robbibt) [inline docs/data/product/dea-surface-reflectance-oa-landsat-9-oli-tirs/index.rst~benji:1]: Is this meant to be here?

File: docs/data/product/dea-waterbodies-landsat/_data.yaml [hunk 10]
```diff
@@ -15,8 +15,10 @@ spatial_data_type: Vector
 time_span:
   start: 1986
   end: Present
-update_frequency: Daily
-next_update: null
+data_updates:
+  frequency: DAILY
```
Comment (robbibt) [inline docs/data/product/dea-waterbodies-landsat/_data.yaml:19]: I _think_ this is updated more like weekly - @BexDunn?

File: docs/data/product/dea-surface-reflectance-landsat-7-etm/_data.yaml (hunk 2 above)
Comment (benji-glitsos-ga) [reply docs/data/product/dea-surface-reflectance-landsat-7-etm/_data.yaml:20]: @robbibt Which of these would make more sense:

```
   "NO_UPDATES": "No further updates",
   "DECOMMISSIONED": "Decommissioned and no further updates",
```

File: docs/data/product/dea-surface-reflectance-landsat-5-tm/_data.yaml (hunk 3 above)
Comment (benji-glitsos-ga) [reply docs/data/product/dea-surface-reflectance-landsat-5-tm/_data.yaml:20]: Ditto the above: NO_UPDATES vs DECOMMISSIONED

File: docs/data/product/dea-surface-reflectance-oa-landsat-9-oli-tirs/index.rst~HEAD
Comment (benji-glitsos-ga) [reply docs/data/product/dea-surface-reflectance-oa-landsat-9-oli-tirs/index.rst~HEAD:1]: Those are weird junk files. I've now deleted them in a different branch: https://github.com/GeoscienceAustralia/dea-knowledge-hub/pull/247

File: docs/data/product/dea-surface-reflectance-landsat-7-etm/_data.yaml (hunk 2 above)
Comment (robbibt) [reply docs/data/product/dea-surface-reflectance-landsat-7-etm/_data.yaml:20]: Plain "Decomissioned" is nice; I think you can probably drop "and no further updates" as it's a bit tautological.

Personally, I kind of like the pairing of "No planned updates" and "Decommisioned" - the first one gives the impression that the product is still valid not but we don't currently intend to modify it (but still gives us the option to if we change our minds). The second one is more final - the product is shut down for good. @CEKrause?

Comment (robbibt) [reply docs/data/product/dea-surface-reflectance-landsat-7-etm/_data.yaml:20]: Or maybe "No further updates" is nice and simple and generic and covers both those cases. Tricky!

File: docs/data/product/dea-waterbodies-landsat/_data.yaml (hunk 10 above)
Comment (benji-glitsos-ga) [reply docs/data/product/dea-waterbodies-landsat/_data.yaml:20]: Thanks - I talked to Bex who said it is Daily.

File: docs/data/product/dea-burnt-area-characteristic-layers-sentinel-2-near-real-time/_data.yaml [hunk 11]
```diff
@@ -3,20 +3,23 @@
...
+  product_type: Derivative
+  spatial_data_type: Raster
 time_span:
   start: 1 Jul 2021
   end: 29 Aug 2023
-update_frequency: Daily
-next_update: No updates planned
+data_updates:
+  frequency: DAILY
+  activity: NO_UPDATES
```
Comment (benji-glitsos-ga) [inline docs/data/product/dea-burnt-area-characteristic-layers-sentinel-2-near-real-time/_data.yaml:21]: @robbibt This product is marked as 'NO_UPDATES', yet it is a Provisional product which intends to be officially released in future. should I instead set mark it as `Paused (updates are temporarily paused)`

Comment (benji-glitsos-ga) [reply docs/data/product/dea-burnt-area-characteristic-layers-sentinel-2-near-real-time/_data.yaml:21]:
```suggestion
  activity: PAUSED
```

File: docs/data/product/daily-satpaths-beta/_data.yaml [hunk 12]
```diff
@@ -3,20 +3,23 @@
...
 header_image: /_files/cmi/SatPaths_image.png
 version: 1.0.3-beta
 is_latest_version: true
 latest_version_link: null
 is_provisional: true
-product_type: Data service
-spatial_data_type: null
+type:
+  product_type: Data service
+  spatial_data_type: null
```
Comment (benji-glitsos-ga) [inline docs/data/product/daily-satpaths-beta/_data.yaml:15]: @robbibt Would you know if this is a Raster or Vector product (or neither)?

File: docs/data/product/australian-national-spectral-database/_data.yaml [hunk 13]
```diff
@@ -3,20 +3,23 @@
...
 header_image: /_files/cmi/spectral_2.jpg
 version: 3.3.0
 is_latest_version: true
 latest_version_link: null
 is_provisional: false
-product_type: Data service
-spatial_data_type: null
+type:
+  product_type: Data service
+  spatial_data_type: null
```
Comment (benji-glitsos-ga) [inline docs/data/product/australian-national-spectral-database/_data.yaml:15]: @robbibt Would you know if this is a Raster or Vector product (or neither)?

File: docs/data/product/daily-satpaths-beta/_data.yaml (hunk 12 above)
Comment (robbibt) [reply docs/data/product/daily-satpaths-beta/_data.yaml:15]: These are services, not data. SatPaths allows you to view the paths of the various satellites we use in DEA to work out when they will be flying overhead.

File: docs/data/product/australian-national-spectral-database/_data.yaml (hunk 13 above)
Comment (robbibt) [reply docs/data/product/australian-national-spectral-database/_data.yaml:15]: Kind of more a service - really a repository or data base into which users can deposit or access spectral information.

File: docs/data/product/daily-satpaths-beta/_data.yaml (hunk 12 above)
Comment (benji-glitsos-ga) [reply docs/data/product/daily-satpaths-beta/_data.yaml:15]: In that case, I'll leave it as `Data service`

**Pull request closed without merging**